src/phoenix/db/models.py

```diff
@@ -133,3 +133,22 @@ async def init_models(engine: AsyncEngine) -> None:
                 description="default project",
             )
         )
+
+
+class SpanAnnotation(Base):
+    __tablename__ = "span_annotations"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    span_rowid: Mapped[int] = mapped_column(ForeignKey("spans.id"))
+    name: Mapped[str]
+    label: Mapped[str]
+    score: Mapped[float]
+    explanation: Mapped[str]
+    meta: Mapped[Dict[str, Any]] = mapped_column(JSON, key="metadata")
+    __table_args__ = (
+        UniqueConstraint(
+            "span_rowid",
+            "name",
+            name="uq_span_annotations_span_rowid_name",
+            sqlite_on_conflict="UPDATE",
```

Comment:
Did not get this right - need to check

Reply:
@mikeldking What case were you trying to cover here? https://www.sqlite.org/lang_conflict.html

Reply:
Making sure subsequent conflict resolution updates the evaluation

Reply:
I think we want `REPLACE` in this case. That will delete the previous row that is causing the conflict on the uniqueness constraint.

> When a [UNIQUE](https://www.sqlite.org/lang_createtable.html#uniqueconst) or [PRIMARY KEY](https://www.sqlite.org/lang_createtable.html#primkeyconst) constraint violation occurs, the REPLACE algorithm deletes pre-existing rows that are causing the constraint violation prior to inserting or updating the current row and the command continues executing normally.

Reply:
Addressed in [14ff266](https://github.com/Arize-ai/phoenix/pull/2788/commits/14ff2663e5a786d92d8837d8d6ba064b096ff337)